src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala

```diff
@@ -26,8 +25,17 @@ trait HttpEventConfiguration extends ScallopConf {
     required = false,
     noshort = true).map(parseHttpEventEndpoints)
 
+  lazy val httpEventCallbackSlowConsumerTimeout = opt[Long]("http_endpoint_slow_consumer_duration",
+    descr = "A http consumer is considered slow, if the delivery takes longer than this timeout (ms)",
```

Comment:
you probably want to adjust this as well, even if it is abstract enough to match callbacks, too, in principle ;)